### PR TITLE
Added FAQ solution on slow python closes ##6935

### PR DESCRIPTION
**Pull request by AdoNunes**
Added in the FAQ the solution to the annoying bug in Mac were running in an IDE such as Spyder or PyCharm commands take very long to execute. 

The issue is solved by changing the system defaults or by installing the appnope module and importing it each time a new console is created. 

closes ##6935

### CODE REVIEW

File: doc/overview/faq.rst [hunk 1]
```diff
@@ -39,6 +39,29 @@ magic ``%gui qt`` after importing MNE/Mayavi/PySurfer should `help
    from mayavi import mlab
    %gui qt
 
+Python runs on MAC extremely slow even on simple commands!
+---------------------------------------------------------
```
Comment (agramfort) [inline doc/overview/faq.rst]:
```suggestion
----------------------------------------------------------
```

Comment (cbrnr) [reply doc/overview/faq.rst]: Replace MAC -> macOS

**Comment by cbrnr** [comment]
Interesting. I just tried this in Spyder and PyCharm on macOS 10.15 but didn't notice the issue. Does this occur only after PyCharm/Spyder has been running for some time?

**Comment by larsoner** [comment]
The [PyPi page](https://pypi.org/project/appnope/) for `appnope` mention macOS 10.9, so it's probably macOS version dependent

**Comment by AdoNunes** [comment]
@cbrnr maybe somehow in your defaults it is already turned off. Sometimes I have to repeat the timing a few times to start having this narcolepsy problem. 

@larsoner My macOs is 10.14.

**Comment by larsoner** [comment]
Okay we should be good here (pushed a commit to adjust indentation):

https://16190-1301584-gh.circle-artifacts.com/0/dev/overview/faq.html#python-runs-on-macos-extremely-slow-even-on-simple-commands

**Comment by larsoner** [comment]
Okay we should be good here (pushed a commit to adjust indentation):

https://16190-1301584-gh.circle-artifacts.com/0/dev/overview/faq.html#python-runs-on-macos-extremely-slow-even-on-simple-commands

**Comment by larsoner** [comment]
Thanks @AdoNunes

**Pull request merged**